prerace_setup.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from animal_factory import factory
+from config import RACERS_CONFIG_PATH
+from constants import *
+
+
+def _initialize_rounds(race_settings) -> dict:
+    rounds = {}
+    for i, curr_round in enumerate(race_settings):
+        racers = _build_round_racers(curr_round, race_settings)
+        _build_round(curr_round, i, race_settings, racers, rounds)
+    return rounds
+
+
+def _build_round(curr_round, i, race_settings, racers, rounds):
```

Comment:
it is better to return a value instead of changing the collection from inside the function